### PR TITLE
Feature: Reprocessing Collections

### PR DESCRIPTION
**Pull request by Femi-Ogunkola**

### CODE REVIEW

File: api/scripts/reprocess_collections.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        logging.info(f'Working on {start} to {end}...')
+        async with database.transaction():
+            query = TokenCollectionsTable.select()
+            query = query.where(TokenCollectionsTable.c.collectionId >= start)
+            query = query.where(TokenCollectionsTable.c.collectionId < end)
+            collectionsToChange = [collection_from_row(row) async for row in database.iterate(query=query)]
+            logging.info(f'Updating {len(collectionsToChange)} collections...')
+            for collection in collectionsToChange:
+                try:
+                    collectionDict = await collectionProcessor.retrieve_collection(address=collection.address)
```
Comment (krishan711) [inline api/scripts/reprocess_collections.py:60]: why is this a dict? the processor returns an object

File: api/scripts/reprocess_collections.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                    if collectionDict:
+                        logging.info(f'Processed: {collection.collectionId}')
+                        await saver.update_collection(collectionId=collection.collectionId, name=collectionDict.get("name"), symbol=collectionDict.get("symbol"), description=collectionDict.get("description"), imageUrl=collectionDict.get("imageUrl"), twitterUsername=collectionDict.get("twitterUsername"), instagramUsername=collectionDict.get("instagramUsername"), wikiUrl=collectionDict.get("wikiUrl"), openseaSlug=collectionDict.get("openseaSlug"), url=collectionDict.get("url"), discordUrl=collectionDict.get("discordUrl"), bannerImageUrl=collectionDict.get("bannerImageUrl"), doesSupportErc721=collectionDict.get("doesSupportErc721"), doesSupportErc1155=collectionDict.get("doesSupportErc1155"))
```
Comment (krishan711) [inline api/scripts/reprocess_collections.py:63]: i dont understand how this is working - the processor should be returning an object. Etiehr way i think in this case we can just use the manager function with shouldForce=True can't we? why do we need to manually save here?

File: api/scripts/reprocess_collections.py (hunk 1 above)
Comment (Femi-Ogunkola) [reply api/scripts/reprocess_collections.py:60]: I just tried to mirror what was existing in the tokenMetadataProcessor

Comment (krishan711) [reply api/scripts/reprocess_collections.py:60]: that script is totlaly different, it gets stuff from the database and edits it

**Review by krishan711 — APPROVED** [review]

**Pull request merged**